.github/rawWorkflows/gh-ci-parameterized-flow.txt

```diff
@@ -85,11 +85,11 @@
           path: ${{ github.job }}-jdk${{ matrix.jdk }}-logs.tar.gz
           retention-days: 30
       - name: Upload test results to BuildPulse for flaky test detection
-        if: github.event_name == 'push' && github.ref == 'refs/heads/main' && !cancelled()
+        if: github.event_name == 'push' && !cancelled()
```

Comment:
We want to remove `github.event_name == 'push'` as well, just `!cancelled()` is perfect